GoogleDataTransport/GDTCORLibrary/GDTCORFlatFileStorage.m

```diff
@@ -206,7 +206,7 @@ - (void)batchWithEventSelector:(nonnull GDTCORStorageEventSelector *)eventSelect
       for (NSString *eventPath in paths) {
         NSError *error;
         GDTCOREvent *event =
-            (GDTCOREvent *)GDTCORDecodeArchive([GDTCOREvent class], eventPath, nil, &error);
+            (GDTCOREvent *)GDTCORDecodeArchiveAtPath([GDTCOREvent class], eventPath, &error);
         if (event == nil || error) {
```

Comment:
The API change will result in the following behavior change:
- The **old** API would return `nil` and **NOT populate** the error if there no resource at the given `eventPath`.
- The **new** API would return `nil` and **populate** the error if there no resource at the given `eventPath`.

So while the error being populated is the only difference, the returned event will still be `nil` so the check on line 210 will cover the case.
```objc 
        if (event == nil || error) {
```